account_cost_center/views/account_cost_center.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+        <!-- Cost Center views -->
+        <record id="account_cost_center_form" model="ir.ui.view">
+            <field name="name">account.cost.center.form</field>
+            <field name="model">account.cost.center</field>
+            <field name="arch" type="xml">
+                <form string="Cost Center" version="7.0">
+                    <sheet>
+                        <group>
+                            <div class="oe_title">
+                                <div class="oe_edit_only">
+                                    <label for="name" string="Title"/>
```

Comment:
It seems this class doesn't have any effect here, I just removed the entire div.